src/blog/migrations/0008_alter_blogpage_video.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.0.4 on 2024-08-01 15:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blog', '0007_blogpage_video'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='blogpage',
+            name='video',
+            field=models.URLField(blank=True, null=True),
+        ),
+    ]
```

Comment:
To make it a bit cleaner, I would merge this migration (0008) and the previous one (0007) into one migration. You can do this in two ways:

## Method 1. (re-do migrations)

```sh
# Revert the last two migrations (go back to 0006)
bin/manage migrate blog 0006

# Delete the last two migrations
rm src/blog/migrations/0008_alter_blogpage_video.py
rm src/blog/migrations/0007_blogpage_video.py

# Re-create migrations
bin/manage makemigrations

# Apply newly created blog migrations
bin/manage migrate blog
```

## Method 2. (squash migrations)

See also: https://docs.djangoproject.com/en/dev/topics/migrations/#squashing-migrations

```sh
# squash the last two migrations with squasmigrations <app> <from> <to>
bin/manage squashmigrations blog 0007 0008
```

Then, after we are sure that the migrations have been applied on the server as well, we can remove the old migrations 0007 & 0008

Reply:
I squashed 0007 and 0008